### ISSUE TITLE
Tooltip improvements

### ISSUE THREAD

**Issue by dovalid** [AgoraSpaceDAO/agora-frontend#60]
Right now the Tooltip children are just wrapped in a Box in the Staked and Platforms components. This is noisy, and not accessible since these divs aren't keyboard focusable, so the tooltips won't be shown if the user navigates using the keyboard. We should use the Tooltip component's shouldWrapChildren prop, but it has a problem right now too: it stays focusable even if the Tooltip is disabled, so it breaks our keyboard navigation flow when the buttons are enabled. I'll open an issue and possibly a PR at Chakra, then we should switch to shouldWrapChildren if it gets merged.
